plugin.video.netzkino_de/resources/settings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <!-- kodion General -->
+    <category label="30000">
+
+        <setting id="kodion.fanart.show" type="bool" label="30021" default="true"/>
+        <setting id="kodion.search.size" type="enum" label="30023" values="0|10|20|30|40|50" default="1" />
+        <setting id="kodion.view.override" type="bool" label="30026" default="false"/>
```

Comment:
To easy the process of fixing what I mentioned before you might want to just remove the options from the plugin settings that might force the addon to change the view. From a quick glance `kodion.view.override`, `kodion.view.default` and `kodion.view.movies` have to be removed